io/las/LasUtils.cpp

```diff
@@ -43,16 +43,20 @@
 namespace pdal
 {
 
+namespace
+{
+    using DT = Dimension::Type;
+    const Dimension::Type lastypes[] = {
+        DT::None, DT::Unsigned8, DT::Signed8, DT::Unsigned16, DT::Signed16,
+        DT::Unsigned32, DT::Signed32, DT::Unsigned64, DT::Signed64,
+        DT::Float, DT::Double
+    };
+}
+
```

Comment:
It's used in both `asType()` and `setType`, and since the `using` is no longer allowed (making it a little noisier) I figured I'd consolidate the init.